src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,11 +1,15 @@
 Microsoft.CodeAnalysis.GeneratorAttribute.GeneratorAttribute(string firstLanguage, params string[] additionalLanguages) -> void
 Microsoft.CodeAnalysis.GeneratorAttribute.Languages.get -> string[]
 Microsoft.CodeAnalysis.ITypeSymbol.IsRecord.get -> bool
+Microsoft.CodeAnalysis.Operations.OperationWalker<TArgument>
```

Comment:
Is making this new API public related to removing laziness from implementation? Feels like this should be kept separate.
If we add a public API, we'd want direct validation on it.